recipes-devtools/cargo/cargo-0.30.0.inc

```diff
@@ -0,0 +1,13 @@
+CARGO_SNAPSHOT = "cargo-0.29.0-${BUILD_ARCH}-unknown-linux-gnu"
```

Comment:
Please use 0.30.0 here as well.

Reply:
Probably using the previous version here is intentional.
It's for building rust/cargo themselves (stage0).

* https://github.com/rust-lang/rust/blob/master/src/stage0.txt
* https://github.com/rust-lang/rust/blob/master/src/bootstrap/README.md

```
# Currently Rust always bootstraps from the previous stable release, and in our
# train model this means that the master branch bootstraps from beta, beta
# bootstraps from current stable, and stable bootstraps from the previous stable
# release.
```

Reply:
Agreed; I didn't see it.

Thanks for the reply on top of mine :+1: